distribution/docs/src/main/resources/templates/appendices.ftl

```diff
@@ -8,7 +8,9 @@ include::${ai.file}[]
 <#list appendixs?sort_by("order") as appendix>
 <#if (ai.children?contains (appendix.parent))>
 
+<#if appendix.title??>
```

Comment:
❓ Why the conditional here?

Reply:
Upstream projects are appending their features onto the "List of Supported Features". Without this here, it was putting the title before each project's list instead of just appending them all together with one title.